src/pages/PlayPage/PlayPage.tsx

```diff
@@ -0,0 +1,87 @@
+import { t } from 'i18next';
+import React, { useState } from 'react';
+
+import { Button } from '../../components/Button';
+import { Card } from '../../components/Card';
+import { ErrorMessage } from '../../components/ErrorMessage';
+import { PageHeader } from '../../components/PageHeader';
+import { useApi } from '../../composables/useApi';
+import { ButtonsContainer, Container, Input } from './PlayPage.styles';
+
+const NUMBER_OF_INPUTS = 6;
```

Comment:
To jedyne miejsce gdzie używasz uppercase do nazwanie zmiennej. Wszędzie indziej używasz pascal, więc tutaj trzymałbym się tego.